CI/unit_tests/models/test_huggingface_flax_model.py

```diff
@@ -0,0 +1,94 @@
+"""
+ZnNL: A Zincwarecode package.
+
+License
+-------
+This program and the accompanying materials are made available under the terms
+of the Eclipse Public License v2.0 which accompanies this distribution, and is
+available at https://www.eclipse.org/legal/epl-v20.html
+
+SPDX-License-Identifier: EPL-2.0
+
+Copyright Contributors to the Zincwarecode Project.
+
+Contact Information
+-------------------
+email: zincwarecode@gmail.com
+github: https://github.com/zincware
+web: https://zincwarecode.com/
+
+Citation
+--------
+If you use this module please cite us with:
+
+Summary
+-------
+"""
+
+import os
+
+os.environ["CUDA_VISIBLE_DEVICES"] = "-1"
```

Comment:
Is this necessary? GitHub as no GPUs so it wont show up here and if you run it locally with a GPU it should theoretically work...

Reply:
No it's not I will remove it

Reply:
Just bumping this in case it was missed.